ldk/javascript/examples/consolidate-loop/Patient.js

```diff
@@ -0,0 +1,98 @@
+class Patient {
```

Comment:
I would research a prettier for js. I see some places you have included semicolons and some you have not.